src/ralph/security/views.py

```diff
@@ -1,6 +1,45 @@
 # -*- coding: utf-8 -*-
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_lazy as _
+
 from ralph.admin.views.extra import RalphDetailView
-from ralph.security.models import SecurityScan
+
+
+class ScanStatusInChangeListMixin(object):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.list_select_related += ['securityscan', ]
```

Comment:
it would not guarantee constant number of queries - each row (each host) will have to check number of vulnerabilities for each scan - each in separated query. We need to make it better :)